data/human/transport missions.txt

```diff
@@ -3582,3 +3582,297 @@ mission "Hauler VI cargo prototype 3"
+			`	When you wake up, spaceport workers on <origin> have just started refueling your ship.`
+				accept
+			label medicineless
+			branch crewless
+				"flagship crew" < 1
+			choice
+				`	(Ask crew members for help.)`
+					goto crew
+				`	(Turn on autopilot.)`
+					goto autopilot
+				`	(Send a distress signal.)`
+					goto distress
+			label crew
+			`	You decide to quickly call on your other crew members to take over the bridge. The few that reply report that they also feeling unwell  and are unable to help.`
+			label crewless
+			choice
+				`	(Turn on autopilot.)`
+					goto autopilot
+				`	(Send a distress signal.)`
+					goto distress
+			label autopilot
+			`	You switch the autopilot on. It goes through a series of troubling beeps, before it realizes that you are trying to land, not hyperspace. At last, "Landing on <origin>" lights up on the screen.`
+			label distress
+			`	Just before you are overcome by the inexplicable sleepiness, you reach over and hit the big "Distress" button. You can only hope that everything works alright.`
+			choice
+				`	(Continue.)`
+			branch poor
+				"net worth" < 30000000
+			`	You find yourself awake in a room, which is definitely not your own. It is lit by a luxurious glass chandelier hanging from a lofty groined ceiling. Looking around, there isn't much furniture, apart from the bed, a few scattered chairs, a row of cabinets to the left, and a large display taking nearly the whole wall in front. The floor is made of white and red marble tiles.`
+				goto doctor
+			label poor
+			`	You find yourself awake in a room, which is definitely not your own. It is lit solely by a few windows that reveal a bustling spaceport nearby. There isn't much furniture apart from the bed and a cabinet.`
+			label doctor
+			`	As you muse and wonder where you are, someone looking like a doctor or perhaps a nurse enters the room.`
+			`	"You are lucky to be still living", she says, "You barely survived an encounter with a recently discovered disease tentatively known as X-3625.".`
+			choice
+				`	"Can you tell me more about this disease?"`
+				`	"Is there anything I should do about it?"`
+					goto do
+				`	"Yep, just like me. Missing every danger through sheer luck."`
+					goto luck
+			`	"The disease appears to be caused by a certain bacteria of unknown origin. It seems to be quite picky about its surroundings. It thrives best in low gravity environments, gradually spreading over the host's body. When exposed to an environment with higher gravity, such as during landing, the bacteria rapidly swell in size, effectively blocking most of the blood flow and causing a sensation of fatigue and sleepiness."`
```

Comment:
This would lore the fact that ships have no (or almost no) artificial gravity.
[Discuss here]

Reply:
If someone has a better reason for the illness to apply only when landing, feel free to comment.

Reply:
How much consistency do we need? Do we care if the player pays attention to the timing? If we don't, we could simply say that it takes a certain amount of time and it was lucky that the player was already landing when it triggered the reaction.

Alternatively, perhaps it is sensitive to changes in magnetic fields. Thus landing on the planet and passing through the magnetic fields trigger a bacterial "bloom."

Reply:
Magnetic fields are a cool idea indeed. Might switch the explanation to that tomorrow-ish if there aren't other better suggestions.